src/main/java/com/byndyusoft/java/spring/microservice/template/client/package-info.java

```diff
@@ -0,0 +1,5 @@
+/**
+ * A place for external integrations in which the service acts as a client.
+ */
+
+package com.byndyusoft.java.spring.microservice.template.client;
```

Comment:
if we add package for cients, maybe add template classes too?

Reply:
Discussed